Wabbajack/View Models/Gallery/ModListGalleryVM.cs

```diff
@@ -173,11 +196,20 @@ public override void Unload()
         private List<string> GetGameTypeEntries()
         {
             List<string> gameEntries = new List<string> { ALL_GAME_TYPE };
-            foreach (var gameType in EnumExtensions.GetAllItems<Game>() )
-            {
-                gameEntries.Add(EnumExtensions.GetDescription<Game>(gameType));
-            }
+            gameEntries.AddRange(EnumExtensions.GetAllItems<Game>().Select(gameType => gameType.GetDescription<Game>()));
             return gameEntries;
         }
+
+        private void UpdateFiltersSettings()
+        {
+            if (!settings.IsPersistent)
+                return;
+            if (!string.IsNullOrEmpty(GameType))
+                settings.Game = GameType;
+            if (Search != null)
+                settings.Search = Search;
```

Comment:
Same with these lines, right now if the current input is null, then the settings will be left with whatever they had in them from before.

But I think rather than worrying too much, I think we can just remove the if statements here?  I don't believe we need to care if things are null and weed those from being saved.